packages/vtree/core/modeling/primitives/cuboid.js

```diff
@@ -0,0 +1,11 @@
+const cuboid = params => {
+  /* const defaults = {
+      size: [1, 1, 1],
+      center: [true, true, true]
+    } */
+  const _params = Object.assign({}, params)
+
+  return Object.assign({}, _params, { type: 'cuboid' })
```

Comment:
Can this assign() be combined with the one above?

Reply:
agreed and done